pom.xml

```diff
@@ -57,7 +57,7 @@
         <guava.version>30.0-jre</guava.version>
         <gson.version>2.8.6</gson.version>
         <slf4j.version>1.7.7</slf4j.version>
-        <cosid.version>1.7.6</cosid.version>
+        <cosid.version>1.8.4</cosid.version>
```

Comment:
@Ahoo-Wang Please update `cosid-core` version in LICENSE file.

Reply:
ok